cmake/nsis/CMakeLists.txt

```diff
@@ -74,6 +73,27 @@ IF(LMMS_HAVE_STK)
 	INSTALL(FILES ${RAWWAVES} DESTINATION "${DATA_DIR}/stk/rawwaves")
 ENDIF()
 
+# Bundle jack
```

Comment:
Should we drop jack bundling? I was sure it was a good idea back then, right now it does look bizare.